ingress/controllers/nginx-third-party/README.md

```diff
@@ -426,6 +426,16 @@ Please check the example `rc-custom-configuration.yaml`
 
 If the Configmap it is updated, NGINX will be reloaded with the new configuration
 
+### NGINX status page
+
```

Comment:
Also make this yaml contain everything needed to start the ingress controller (i.e includs default svc+rc etc)